src/main.ts

```diff
@@ -69,6 +69,7 @@ async function start () {
         themes: {
           light: {
             primary: '#1669bb', // same as $app-blue
+            secondary: '#38598a', // Same as $p-blue-text
```

Comment:
Normally "secondary" is a gray colour (as in, "it's not the primary action").
Re: https://vuetifyjs.com/en/features/theme/#customizing

Does this change break any existing use of secondary?

Reply:
Not that i noticed, what do you think a good name would be for a non-clickable, darker blue.

Reply:
"app-dk-blue" ?

It just seems like "secondary" may be used for something different.

This project isn't too bad for colour names, but the idea is to have a small list of colours ("theme").

Yes, the fact that it's for something clickable could be part of the name. Yes, "app-xxx" is very generic, possibly overly generic, but I can't think of a better colour framework atm.

Reply:
> Not that i noticed

Did you search for other uses of "secondary". If not, you could hijack it I guess. (And update any other places that should use it.)

Reply:
kk, updated in next commit